templates/repo/issue/view_content.tmpl

```diff
@@ -166,21 +166,23 @@
 
 <template id="issue-comment-editor-template">
 	<div class="ui comment form">
+		<div class="field">
 		{{template "shared/combomarkdowneditor" (dict
 			"locale" $.locale
 			"MarkdownPreviewUrl" (print .Repository.Link "/markup")
 			"MarkdownPreviewContext" .RepoLink
 			"TextareaName" "content"
 			"DropzoneParentContainer" ".ui.form"
 		)}}
+		</div>
```

Comment:
I do not think it is a must. A lot of old code doesn't have such `indent` either, and this block is pretty simple.

Without `Indent`, the diff is more clear for reviewers.

Reply:
We shouldn't optimize for diff reading.

Reply:
```html
<div ...>
{{template ....}}
</div>
```

Without indent, it is still clear and readable, right?

Reply:
Yes, but lack of indent irks me, but whatever.

Reply:
If you want to make it easier to review the diffs, you do all your changes without fixing indentation, and then add a final commit that fixes indentation only.  Optionally, you may want to note in the PR description that you've done this so that reviewers can look at it either with or without indentation fixes, as their choice.

Reply:
Yeah, could do that. Ideally we would have a auto formatter, but the ecosystem around go templates is weak unfortunately. There is a minimal brace fixer during `make fmt`, but we could certainly do more there.

Reply:
> If you want to make it easier to review the diffs, you do all your changes without fixing indentation, and then add a final commit that fixes indentation only. Optionally, you may want to note in the PR description that you've done this so that reviewers can look at it either with or without indentation fixes, as their choice.

Usually I provide a link like this:

![image](https://user-images.githubusercontent.com/2114189/233754359-579d9ac9-7690-4755-a42c-13bce051f610.png)